data/api/client-server/support.yaml

```
@@ -22,9 +22,12 @@ paths:
       description: |-
         Gets server admin contact and support page of the domain.
 
+        {{% boxes/note %}}
         Like the [well-known discovery URI](/client-server-api/#well-known-uri),
```

I guess I need to check for this...

```suggestion
        Like the [well-known discovery URI](/client-server-api/#well-known-uris),
```